protoc-gen-go/generator/generator.go

```diff
@@ -1343,7 +1342,7 @@ func (g *Generator) generateHeader() {
 	if !haveImportPath {
 		g.P("package ", name)
 	} else {
-		g.P("package ", name, " // import ", strconv.Quote(g.ImportPrefix+string(importPath)))
+		g.P("package ", name, " // import ", g.ImportPrefix+importPath)
```

Comment:
Why is this `path.Join` in the `grpc` package but a straight concatenation here?

Reply:
Because ImportPrefix is completely broken.

Reply:
That was harsh, so let me expand.

The fact that import_prefix rewrites standard imports (proto, grpc, context) makes no sense. The fact that it rewrites them inconsistently is almost irrelevant, since nobody wanted that rewriting in the first place.

We could try to fix it, but everything that it does is done better and more reasonably by the `go_package` option.